src/psyclone/tests/core/symbolic_maths_test.py

```diff
@@ -268,3 +269,40 @@ def test_symbolic_math_use_reserved_names(fortran_reader, expressions):
     schedule = psyir.children[0]
     sym_maths = SymbolicMaths.get()
     assert sym_maths.equal(schedule[0].rhs, schedule[1].rhs) is True
+
+
+@pytest.mark.parametrize("expr,expected", [
+    ("1.0", "1.0"),
+    ("a", "a"),
+    ("a*b+c", "a * b + c"),
+    ("c+a*b", "c + a * b"),
+    ("(a*b)+c", "a * b + c"),
+    ("a*(b+c)", "a * b + a * c"),
+    ("a*((b+c)/d)", "a * b / d + a * c / d")])
```

Comment:
Please could you add an example with array accesses.

Reply:
OK.